content/20.pharmaceuticals.md

```diff
@@ -489,58 +486,59 @@ As a result, the trial was terminated early, offering additional evidence agains
 Thus, the administration of dexamethasone for COVID-19 is likely to simulate the release of glucocorticoids endogenously during stress, resulting in binding of the synthetic steroid to the glucocorticoid receptor and the associated inhibition of the production of proinflammatory cytokines.
-In this model, dexamethasone reduces inflammation by stimulating the mechanism that reduces inflammation following a threat such as immune challenge.
+In this model, dexamethasone reduces inflammation by stimulating the biological mechanism that reduces inflammation following a threat such as immune challenge.
 Immunosuppressive drugs such as steroids are typically contraindicated in the setting of infection [@doi:10.1177/2040622313485275], but because COVID-19 results in hyperinflammation that appears to contribute to mortality via lung damage, immunosuppression may be a helpful approach to treatment [@doi:10/ggnzmc].
 The decision of whether and/or when to counter hyperinflammation with immunosuppression in the setting of COVID-19 was an area of intense debate, as the risks of inhibiting antiviral immunity needed to be weighed against the beneficial anti-inflammatory effects [@doi:10/ggq8hs].
 As a result, guidelines early in the pandemic typically recommended avoiding treating COVID-19 patients with corticosteroids such as dexamethasone [@doi:10/ggks86].
 
 The application of dexamethasone for the treatment of COVID-19 was evaluated as part of the multi-site RECOVERY trial in the United Kingdom [@doi:10.1101/2020.06.22.20137273].
-Over 6,000 hospitalized COVID-19 patients were assigned into standard care or treatment (dexamethasone) arms of the trial with a 2:1 ratio.
+Over 6,000 hospitalized COVID-19 patients were assigned into the SOC or treatment (dexamethasone) arms of the trial with a 2:1 ratio.
 At the time of randomization, some patients were ventilated (16%), others were on non-invasive oxygen (60%), and others were breathing independently (24%).
 Patients in the treatment arm were administered dexamethasone either orally or intravenously at 6 mg per day for up to 10 days.
 The primary end-point was the patient's status at 28-days post-randomization (mortality, discharge, or continued hospitalization), and secondary outcomes analyzed included the progression to invasive mechanical ventilation over the same period.
-The 28-day mortality rate was found to be lower in the treatment group than in the standard care group (21.6% vs 24.6%, p <0.001).
+The 28-day mortality rate was found to be lower in the treatment group than in the SOC group (21.6% vs 24.6%, p <0.001).
```

Comment:
p in italics? Saw that in other parts of the manuscript

Reply:
```suggestion
The 28-day mortality rate was found to be lower in the treatment group than in the SOC group (21.6% vs 24.6%, _p_ <0.001).
```